### PR TITLE
CIV-7439 info and relationships

### PR DESCRIPTION
**Pull request by laarmada**
### JIRA link ###

https://tools.hmcts.net/jira/browse/CIV-7439

### Change description ###

Info and Relationships (Level A): Convert H2 to H1 and H3 to H2

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/main/views/features/claim/yourDetails/claimant-organisation-details.njk [hunk 1]
```diff
@@ -46,8 +46,7 @@
       {{ t('PAGES.COMPANY_DETAILS.ADDRESS_FOR_SERVICE') }}</p>
     {{ postcodeLookupAndAddressForm(t, {
       party: party,
-      addressType:'correspondenceAddress',
-      addressTitle:addressTitle
+      addressType:'correspondenceAddress'
```
Comment (GorilaNaranja) [inline src/main/views/features/claim/yourDetails/claimant-organisation-details.njk:49]: Don't we need "addressTitle" anymore here?

**Review by LeonardoPalmeiro — DISMISSED** [review]

File: src/main/views/features/mediation/i-dont-want-free-mediation.njk [hunk 2]
```diff
@@ -21,11 +22,17 @@
...
       <div class="govuk-grid-row">
         <div class="govuk-grid-column-two-thirds">
-          <h1 class="govuk-heading-l">{{ t('PAGES.I_DON_T_WANT_FREE_MEDIATION.DO_NOT_AGREE_TO_FREE_MEDIATION') }}</h1>
-          {% set ttext = t('PAGES.I_DON_T_WANT_FREE_MEDIATION.YOU_HAVE_CHOSEN_TELL_US_WHY') %}
-          {{govukHint({text: ttext})}}
           <form method="post" novalidate>
             {{ csrfProtection(csrf) }}
+            {% call govukFieldset({
+               legend: {
+                 html: t('PAGES.I_DON_T_WANT_FREE_MEDIATION.DO_NOT_AGREE_TO_FREE_MEDIATION'),
```
Comment (GorilaNaranja) [inline src/main/views/features/mediation/i-dont-want-free-mediation.njk:29]: Instead use "html" we can use "text" if it is only a translation

File: src/main/views/features/mediation/i-dont-want-free-mediation.njk [hunk 3]
```diff
@@ -21,11 +22,17 @@
...
+                 classes: "govuk-fieldset__legend--l",
+                 isPageHeading: true
+               }
+            }) %}
+            {% set ttext = t('PAGES.I_DON_T_WANT_FREE_MEDIATION.YOU_HAVE_CHOSEN_TELL_US_WHY') %}
```
Comment (GorilaNaranja) [inline src/main/views/features/mediation/i-dont-want-free-mediation.njk:34]: I will rename it to "text" with one t

File: src/main/views/features/mediation/mediation-disagreement.njk [hunk 4]
```diff
@@ -18,18 +19,22 @@
...
       <div class="govuk-grid-row">
         <div class="govuk-grid-column-two-thirds">
-
           <div class="govuk-!-margin-bottom-8">
-            <h1 class="govuk-heading-l">{{ t('PAGES.MEDIATION_DISAGREEMENT.MEDIATION_DISAGREEMENT') }}</h1>
+            <form method="post" novalidate>
+             {{ csrfProtection(csrf) }}
+                {% call govukFieldset({
+                  legend: {
+                     html: t('PAGES.MEDIATION_DISAGREEMENT.MEDIATION_DISAGREEMENT'),
```
Comment (GorilaNaranja) [inline src/main/views/features/mediation/mediation-disagreement.njk:32]: same here, use text instead html

File: src/main/views/features/response/admission/payment-date.njk [hunk 5]
```diff
@@ -28,7 +29,13 @@
       {% endif %}
       <div class="govuk-grid-row">
         <div class="govuk-grid-column-two-thirds">
-          <h1 class="govuk-heading-l">{{ t(title) }}</h1>
+          {% call govukFieldset({
+             legend: {
+                html: t(title),
```
Comment (GorilaNaranja) [inline src/main/views/features/response/admission/payment-date.njk:34]: text

File: src/main/views/features/response/citizenDob/citizen-dob.njk [hunk 6]
```diff
@@ -17,25 +18,26 @@
...
-          </h1>
-        </div>
-      </div>
       <div class="govuk-grid-row">
         <div class="govuk-grid-column-two-thirds">
           <form method="post" novalidate>
             {{ csrfProtection(csrf) }}
+              {% call govukFieldset({
+                 legend: {
+                    html: title,
```
Comment (GorilaNaranja) [inline src/main/views/features/response/citizenDob/citizen-dob.njk:36]: text

File: src/main/views/features/response/statementOfMeans/courtOrders/court-orders.njk [hunk 7]
```diff
@@ -22,11 +23,16 @@
...
       <div class="govuk-grid-row">
         <div class="govuk-grid-column-two-thirds">
-          <h1 class="govuk-heading-l">{{ t('PAGES.COURT_ORDERS.TITLE') }}</h1>
           <div data-module="moj-add-another">
             <form method="post" novalidate>
               {{ csrfProtection(csrf) }}
-
+            {% call govukFieldset({
+             legend: {
+                html: t('PAGES.COURT_ORDERS.TITLE'),
```
Comment (GorilaNaranja) [inline src/main/views/features/response/statementOfMeans/courtOrders/court-orders.njk:31]: text

File: src/main/views/features/response/statementOfMeans/debts/debts.njk [hunk 8]
```diff
@@ -118,30 +119,26 @@
...
-      </div>
-      <form method="post" novalidate>
-        {{ csrfProtection(csrf) }}
-        <div class="govuk-grid-row">
-          <div class="govuk-grid-column-full">
+          <form method="post" novalidate>
+            {{ csrfProtection(csrf) }}
+              {% call govukFieldset({
+                 legend: {
+                 html: title,
```
Comment (GorilaNaranja) [inline src/main/views/features/response/statementOfMeans/debts/debts.njk:126]: text

File: src/main/views/features/response/statementOfMeans/dependants/dependants.njk [hunk 9]
```diff
@@ -25,10 +26,15 @@
       {% endif %}
       <div class="govuk-grid-row">
         <div class="govuk-grid-column-two-thirds">
-          <h1 class="govuk-heading-l">{{ t('PAGES.DEPENDANTS.TITLE') }}</h1>
           <form method="post" novalidate>
             {{ csrfProtection(csrf) }}
-
+            {% call govukFieldset({
+             legend: {
+                html: t('PAGES.DEPENDANTS.TITLE'),
```
Comment (GorilaNaranja) [inline src/main/views/features/response/statementOfMeans/dependants/dependants.njk:33]: text

File: src/main/views/features/response/statementOfMeans/residence.njk [hunk 10]
```diff
@@ -19,10 +20,15 @@
       {% endif %}
       <div class="govuk-grid-row">
         <div class="govuk-grid-column-two-thirds">
-          <h1 class="govuk-heading-l">{{ t('PAGES.RESIDENCE.TITLE') }}</h1>
           <form method="post" novalidate>
             {{ csrfProtection(csrf) }}
-
+            {% call govukFieldset({
+             legend: {
+                html: t('PAGES.RESIDENCE.TITLE'),
```
Comment (GorilaNaranja) [inline src/main/views/features/response/statementOfMeans/residence.njk:27]: text

File: src/main/views/features/mediation/mediation-disagreement.njk [hunk 11]
```diff
@@ -18,18 +19,22 @@
...
       {% if form.hasErrors() %}
-        {{ errorSummary(errors, t) }}
+        {{ errorSummary(form.getErrors(), t) }}
       {% endif %}
       <div class="govuk-grid-row">
         <div class="govuk-grid-column-two-thirds">
-
           <div class="govuk-!-margin-bottom-8">
-            <h1 class="govuk-heading-l">{{ t('PAGES.MEDIATION_DISAGREEMENT.MEDIATION_DISAGREEMENT') }}</h1>
+            <form method="post" novalidate>
```
Comment (laperla11) [inline src/main/views/features/mediation/mediation-disagreement.njk:28]: as per doumentation, it's not required to use the `fieldset` for `radios, checkboxes and date input`, so you can use `fieldset` attribute of those components.

**Review by laperla11 — DISMISSED** [review]

**Review by GorilaNaranja — APPROVED** [review]

File: src/main/views/features/claim/yourDetails/claimant-organisation-details.njk (hunk 1 above)
Comment (laarmada) [reply src/main/views/features/claim/yourDetails/claimant-organisation-details.njk]: thanks, done

File: src/main/views/features/mediation/i-dont-want-free-mediation.njk (hunk 2 above)
Comment (laarmada) [reply src/main/views/features/mediation/i-dont-want-free-mediation.njk]: done

File: src/main/views/features/mediation/i-dont-want-free-mediation.njk (hunk 3 above)
Comment (laarmada) [reply src/main/views/features/mediation/i-dont-want-free-mediation.njk]: done

File: src/main/views/features/mediation/mediation-disagreement.njk (hunk 4 above)
Comment (laarmada) [reply src/main/views/features/mediation/mediation-disagreement.njk]: done

File: src/main/views/features/response/admission/payment-date.njk (hunk 5 above)
Comment (laarmada) [reply src/main/views/features/response/admission/payment-date.njk]: done

File: src/main/views/features/response/statementOfMeans/courtOrders/court-orders.njk (hunk 7 above)
Comment (laarmada) [reply src/main/views/features/response/statementOfMeans/courtOrders/court-orders.njk]: done

File: src/main/views/features/response/statementOfMeans/debts/debts.njk (hunk 8 above)
Comment (laarmada) [reply src/main/views/features/response/statementOfMeans/debts/debts.njk]: done

File: src/main/views/features/response/statementOfMeans/residence.njk (hunk 10 above)
Comment (laarmada) [reply src/main/views/features/response/statementOfMeans/residence.njk]: done

File: src/main/views/features/mediation/i-dont-want-free-mediation.njk (hunk 3 above)
[the remark this answers is not in the document]
Comment (laarmada) [reply src/main/views/features/mediation/i-dont-want-free-mediation.njk]: I agree I have implemented that one.

File: src/main/views/features/mediation/mediation-disagreement.njk (hunk 11 above)
Comment (laarmada) [reply src/main/views/features/mediation/mediation-disagreement.njk:28]: I didnt see that it´s include into the yesNo macro, I have removed in all related...

**Review by GorilaNaranja — DISMISSED** [review]

**Review by LeonardoPalmeiro — APPROVED** [review]

**Review by LeonardoPalmeiro — DISMISSED** [review]

**Review by GorilaNaranja — DISMISSED** [review]

**Review by LeonardoPalmeiro — APPROVED** [review]

**Pull request merged**